integration/configs/dmsgptyhost1.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
Can we name them to indicate they are windows specific configs?

Reply:
ah ok, I will do so.